### PR TITLE
Quality check post p rocess

### PR DESCRIPTION
**Pull request by SmariF89**

### CODE REVIEW

File: GetRecordings/modules/extractor.py [hunk 1]
```diff
@@ -102,20 +102,20 @@ def update_metadata(self):
         print(f'Metadata entries: {len(metadata)}')
         print(f'Database entries: {len(is_valid_new)}')
 
-        update_count:int = 0
-        for id in tqdm(metadata.index):
-            prev_value = metadata.at[id, 'is_valid']
-            new_value = str(is_valid_new.at[int(id), 'is_valid'])
+        with open('update_log.txt', 'a') as f_out:
```
Comment (SmariF89) [inline GetRecordings/modules/extractor.py:105]: Moved open up here so that we don't have to open and close it thousands of times xD

**Comment by judyfong** [comment]
Some points, mostly regarding your document:


	
1. btw noisy data is good for asr because it better simulates a real world environment. it's bad for tts data though
	
2. english uses commas for numbers such as 791,104.
	
3. how long is this document? # of pages? can you reduce it a bit? if needed, i can give you tips of where to cut
	
3. for the two scripts you should link to the files
	
4. you should cite sources:
	
```

		
McAuliffe, Michael, Michaela Socolof, Sarah Mihuc, Michael Wagner, and Morgan Sonderegger (2017). Montreal Forced Aligner [Computer program]. Version 0.9.0, retrieved 17 January 2017 from http://montrealcorpustools.github.io/Montreal-Forced-Aligner/.
```
		
```
@inproceedings{gunason17_interspeech,
  author={Jón Guðnason and Matthías Pétursson and Róbert Kjaran and Simon Klüpfel and Anna Björk Nikulásdóttir},
  title={{Building ASR Corpora Using Eyra}},
  year=2017,
  booktitle={Proc. Interspeech 2017},
  pages={2173--2177},
  doi={10.21437/Interspeech.2017-1352}
}
```	
	
	
	
	
6. markdown format for links is []() not []<>
	
12. the author list should have extra lines between author names OR have it as a list with asterisks
	
13. use more precise wording like gunnar said and add the responses you had to his questions

**Comment by staffanru** [comment]
Code looks clean and well documented with comments.

File: QualityCheckPostProcess/scripts/generate_marosijo_votes.py [hunk 2]
```diff
@@ -0,0 +1,231 @@
+"""
```
Comment (SmariF89) [inline QualityCheckPostProcess/scripts/generate_marosijo_votes.py:1]: This should be removed.

**Comment by SmariF89** [comment]
> Some points, mostly regarding your document:
> 
>     1. btw noisy data is good for asr because it better simulates a real world environment. it's bad for tts data though
> 
>     2. english uses commas for numbers such as 791,104.
> 
>     3. how long is this document? # of pages? can you reduce it a bit? if needed, i can give you tips of where to cut
> 
>     4. for the two scripts you should link to the files
> 
>     5. you should cite sources:
> 
> 
> ```
> 
> 		
> McAuliffe, Michael, Michaela Socolof, Sarah Mihuc, Michael Wagner, and Morgan Sonderegger (2017). Montreal Forced Aligner [Computer program]. Version 0.9.0, retrieved 17 January 2017 from http://montrealcorpustools.github.io/Montreal-Forced-Aligner/.
> ```
> 
> ```
> @inproceedings{gunason17_interspeech,
>   author={Jón Guðnason and Matthías Pétursson and Róbert Kjaran and Simon Klüpfel and Anna Björk Nikulásdóttir},
>   title={{Building ASR Corpora Using Eyra}},
>   year=2017,
>   booktitle={Proc. Interspeech 2017},
>   pages={2173--2177},
>   doi={10.21437/Interspeech.2017-1352}
> }
> ```
> 
>     1. markdown format for links is  not []<>
> 
>     2. the author list should have extra lines between author names OR have it as a list with asterisks
> 
>     3. use more precise wording like gunnar said and add the responses you had to his questions

I will look into this next week :)

File: QualityCheckPostProcess/readme.md [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+
+## The tools
+This package contains the tools that were used to process the Marosijo output. It outputs a list of votes which are ready to be put into the Samrómur database.
+
+### generate_marosijo_votes.py
+This is the main utility for Marosijo score processing and vote generation. You give a path to a metadata file where the marosijo_score column has been filled with values, a path to a list of Montreal Forced Aligner unalignable recordings and some filenames for the output files.
+
+The output consists of two files:
```
Comment (SmariF89) [inline QualityCheckPostProcess/readme.md:223]: I only mention one of the files. I forgot to write about the statistics file. Should add a few line about it.

**Review by staffanru — APPROVED** [review]
Approved when concerns regarding the report has been attended to.

**Comment by SmariF89** [comment]
I also remember that I have some Venn diagrams which explain some of the statistics even better. I should add those.

**Review by judyfong — APPROVED** [review]
Good changes to the report

**Pull request merged**